app/components/Sidebar/Main.js

```diff
@@ -123,23 +123,25 @@ function MainSidebar() {
                   documents.active ? documents.active.template : undefined
                 }
               />
-              <SidebarLink
-                to="/drafts"
-                icon={<EditIcon color="currentColor" />}
-                label={
-                  <Drafts align="center">
-                    {t("Drafts")}
-                    <Bubble count={documents.totalDrafts} />
-                  </Drafts>
-                }
-                active={
-                  documents.active
-                    ? !documents.active.publishedAt &&
-                      !documents.active.isDeleted &&
-                      !documents.active.isTemplate
-                    : undefined
-                }
-              />
+              {can.createDocument && (
```

Comment:
Let's also hide the "Templates" item if you can't create a document, after testing it out locally it felt odd to see that, a bit like being "behind the scenes".